c++/include/orc/Writer.hh

```diff
@@ -262,6 +262,17 @@ namespace orc {
      * @return if not set, the default is false
      */
     bool getUseTightNumericVector() const;
+
+     /**
```

Comment:
nit. Indentation?

Reply:
This is detected by the CI format check.

Reply:
Fixed.